src/engines/nwn2/script/functions.h

```diff
@@ -246,6 +246,8 @@ class Functions {
 	void getSkillRank   (Aurora::NWScript::FunctionContext &ctx);
 	void getHasFeat     (Aurora::NWScript::FunctionContext &ctx);
 
+	void featAdd        (Aurora::NWScript::FunctionContext &ctx);
```

Comment:
If you're separating it by a blank link, you can drop that extra alignment in front of the opening paren there. It's supposed to line up each sub-group.